spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -147,6 +174,15 @@
       expect(response).to redirect_to spree.checkout_state_path(:address)
     end
 
+    context 'and not deafult locale' do
+      it 'redirects to the checkout_path after saving with non default locale' do
+        allow(order).to receive(:update) { true }
```

Comment:
Do we need the return value "true" here?

Reply:
Hmm, it doesn't seem necessary, I just assumed it is, becouse that's how it was in test for default locale.